src/i18n/sv.js

```diff
@@ -0,0 +1,102 @@
+export default{
+	error : {
+		forgotPassword : {
+			"too_many_requests" : "Du har nått gränsen för maximalt antal försök att ändra ditt lösenord. Var vänlig vänta innan du försöker igen.",
+			"lock.fallback" : "Något gick fel när vi försökte ändra ditt lösenord."
+		},
+		login : {
+			"blocked_user" : "Användaren är spärrad.",
+			"invalid_user_password" : "Inloggningsuppgifterna är felaktiga.",
+			"lock.fallback" : "Något gick fel vid inloggningsförsöket.",
+			"lock.invalid_code" : "Fel kod.",
+			"lock.invalid_email_password" : "Fel e-postadress eller lösenord.",
+			"lock.invalid_username_password" : "Fel användarnamn eller lösenord.",
+			"lock.network" : "Kunde inte kontakta servern. Var vänlig kontrollera din uppkoppling och försök igen.",
+			"lock.popup_closed" : "Popup-fönster stängt. Försök igen.",
+			"lock.unauthorized" : "Åtkomst nekad. Försök igen.",
+			"password_change_required" : "Du behöver ändra ditt lösenord då detta är första gången du loggar in eller för att ditt lösenord har gått ut.", // TODO: verify error code
+			"password_leaked" : "Din konto har blivit låst då ditt lösenord har lagts upp på en annan webbsida. Du har fått e-post med instruktioner om hur du låser upp ditt konto.",
+			"too_many_attempts" : "Ditt konto har blivit låst efter att flera felaktiga inloggningsförsök har gjorts."
+		},
+		passwordless : {
+			"bad.email" : "Din e-postadress är ogiltig.",
+			"bad.phone_number" : "Ditt telefonnummer är ogiltigt.",
+			"lock.fallback" : "Något gick fel."
+		},
+		signUp : {
+			"invalid_password" : "Lösenordet är ogiltigt.",
+			"lock.fallback" : "Något gick fel vid registreringen.",
+			"password_dictionary_error" : "Lösenordet är för vanligt.",
+			"password_no_user_info_error" : "Lösenordet baseras på personuppgifter.",
+			"password_strength_error" : "Lösenordet är för svagt.",
+			"user_exists" : "Användaren finns redan.",
+			"username_exists" : "Användarnamnet finns redan."
+		}
+	},
+	success : { // success messages show above the form or in a confirmation pane
+		logIn : "Tack för att du loggar in.",
+		forgotPassword : "Vi har skickat dig ett e-postmeddelande där du kan återställa ditt lösenord.",
+		magicLink : "Vi har skickat en länk för att logga in<br />till %s.",
+		signUp : "Tack för att du registrerade dig."
+	},
+	blankErrorHint : "Får inte lämnas tom",
+	codeInputPlaceholder : "Din kod",
+	databaseEnterpriseLoginInstructions : "",
+	databaseEnterpriseAlternativeLoginInstructions : "eller",
+	databaseSignUpInstructions : "",
+	databaseAlternativeSignUpInstructions : "eller",
+	emailInputPlaceholder : "dinepostadress@exempel.se",
+	enterpriseLoginIntructions : "Logga in med dina företagsuppgifter.",
+	enterpriseActiveLoginInstructions : "Ange dina företagsuppgifter för %s.",
```

Comment:
I'm not sure of the context here.  The English term says "Please enter your coorporate credentials at %s.".  If "at" in this context indicates a place to enter the credentials, it should be "vid" rather than "för".  Otoh, if "at" indicates the company name, "för" works fine.